1-3.md

```diff
@@ -63,13 +66,14 @@ The following topics help achieve the learning outcomes. Adapt the sequence of t
 
 {% include excol.html type="middle" %}
 
-Explain how accessibility helps organizations remove barriers.  Compare digital barriers with those which are social or architectural. Use the resource [Business Case for Digital Accessibility (Drive Innovation)](https://www.w3.org/WAI/business-case/#drive-innovation).
```

Comment:
Don't think "how accessibility removes barriers" fits under the heading of "innovation driver". Also the comparing does not really fit well here -- I moved it down into "broader benefits"